src/main/java/entities/CollaborativeTask.java

```diff
@@ -0,0 +1,278 @@
+package entities;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+
+public class CollaborativeTask extends Task implements Timeblockable {
+    private boolean recurring;
+    private String frequency;
+    private LocalDateTime startTime;
+    private LocalDateTime endTime;
+    private LocalDateTime deadline;
+    private ArrayList<ArrayList<LocalDateTime>> timeBlocks;
+    private ArrayList<StudentUser> teammates;
+    private ArrayList<StudentUser> pendingTeammates;
+    private ArrayList<StudentUser> declinedTeammates;
+    private StudentUser leader;
+
+    /**
+     * Create a CollaborativeTask with a title, priority, and deadline
+     * If the collaborative task is recurring, indicate the frequency (eg "daily", "weekly", "monthly)
+     * Otherwise, the frequency is blank
+     *
```

Comment:
I think priority is missing as a param on this comment

If its there and I just missed it then disregard my comment

Otherwise, just change after merging